src/resolvers/vueuse.ts

```diff
@@ -0,0 +1,11 @@
+import { ComponentResolver } from '../types'
+
+/**
+ * Resolve for VueUse Components
+ * 
+ * @link https://github.com/vueuse/vueuse
+ */
+export const VueUseComponentsResolver = (): ComponentResolver => (name: string) => {
+  if (name.toLowerCase().startsWith('use') || name.toLowerCase().startsWith('on'))
```

Comment:
```suggestion
  if (/^(?:Use|On)[A-Z]/.test(name))
```

Tho I still thinking maybe we should read the `indexes.json` to have more accurate match.

Reply:
Oh yeah that would be a much better way I think. Any preferred way we should go about reading `indexes.json`?

Reply:
Maybe we should ship it along with the package

Reply:
This should do in 5.0.0-beta.3:

```js
require('@vueuse/core/indexes.json')
```